components/feature/qr/src/main/java/mozilla/components/feature/qr/QrFeature.kt

```diff
@@ -37,7 +37,7 @@ class QrFeature(
 ) : LifecycleAwareFeature, BackHandler {
     private var containerViewId: Int = 0
 
-    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    @VisibleForTesting
```

Comment:
Why did you remove the parameter here? Because it's the default?

Reply:
Yep!

Reply:
I often still use "private" in order to:
* Make it more readable without having to know or lookup the default
* Make it clear what my intent is - as opposed to "I forgot or didn't know there's a parameter"

What do you think?

Reply:
Tbh, I prefer simple `@VisibleForTesting` in this case. Private-by-default is the most strict behavior, so we'll not expose part of the private API accidentally. And sources looks more readable - properties are parsed visually with less effort.

Anyway, as a code owner you always can insist and I'll change it back. :sweat_smile: 

BTW. What do you think about project-level code style document? I can prepare some preview version that can be extended when new questions will arise.

Reply:
@pocmo What kind of practice I should consider as a project guideline for now?

Reply:
We likely aren't consistent here, but I would let it in as well.

Reply:
:ok: